app/views/account/my_tenants/_form.html.slim

```diff
@@ -0,0 +1,6 @@
+= simple_form_for current_user.tenant, url: account_my_tenant_path,
```

Comment:
change to simple_form_for  `@tenant`

Reply:
but it doesn't work... 
![simple_form_issue](https://user-images.githubusercontent.com/28875500/34521355-00474f08-f096-11e7-84f6-d239a5bbd040.png)